embedx/config.schema.json

```diff
@@ -1963,9 +1963,29 @@
             }
           },
           "additionalProperties": false
+        },
+        "channels": {
+          "type": "array",
+          "items": {
+            "title": "Courier channel configuration",
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "title": "Channel id",
+                "description": "The channel id. Corresponds to the .via property of the identity schema for recovery, verification, etc. Currently only phone is supported.",
+                "maxLength": 255,
+                "enum": ["phone"]
```

Comment:
Shouldn't this be `sms` given that we're not calling anyone?

Reply:
It's essentially an HTTP based sender, and you can use e.g. Twilio to dispatch phone calls instead of SMS here already, AFAICT (have only tried SMS so far). So `phone` makes most sense to me.

Reply:
It looks like phone calls are a bit more difficult as they have more states (calling, ringing, answered, done) and more instructions including a callback for when the call is done (see: https://www.twilio.com/docs/voice/make-calls#initiate-an-outbound-call-with-twilio), as well as a more complex set up for what twilio should do when making the call (https://www.twilio.com/docs/voice/twiml). I'm not sure if this would work with the way we do it here.

I think phone and sms are interchangeable too. SMS is probably easier to understand what it does for people who need to configure/implement this, than `phone`. We could also call it `http_callback` (what it really is) but I think that would be even more difficult to understand. Up to you

Reply:
I changed the value for the `verification.via` key to `"sms"`, because we also call this message type `"sms"` in the code. That should make it easier to understand for now. We'll have to figure out, if this is an issue, if users want to use whatsapp or other messengers here.

Reply:
Actually nvm. We have a few places in the code that already make assumptions about the verifiable address if it's via value is "phone". Untangling that, is pretty difficult. 

E.g. https://github.com/ory/kratos/blob/c1406fb79fce56f6da9917d50b41065ae5a1c9bf/identity/extension_verification.go#L45